seed/lib/mappings/data/pm-mapping.json

```diff
@@ -152,6 +242,69 @@
     "type": "float",
     "schema": ""
   },
+  {
+    "display_name": "Geocoding Confidence",
+    "to_field": "geocoding_confidence",
+    "to_table_name": "PropertyState",
+    "from_field": "Geocoding Confidence",
+    "units": "",
+    "type": "string",
+    "schema": ""
+  },
+  {
+    "display_name": "Home Energy Score ID",
+    "to_field": "home_energy_score_id",
+    "to_table_name": "PropertyState",
+    "from_field": "Home Energy Score ID",
+    "units": "",
+    "type": "string",
+    "schema": ""
+  },
+  {
+    "display_name": "Jurisdiction Property Lot ID",
+    "to_field": "jurisdiction_property_lot_id",
+    "to_table_name": "PropertyState",
+    "from_field": "Jurisdiction Property Lot ID",
+    "units": "",
+    "type": "string",
+    "schema": ""
+  },
+  {
+    "display_name": "Latitude",
+    "to_field": "latitude",
+    "to_table_name": "PropertyState",
+    "from_field": "Latitude",
+    "units": "",
+    "type": "float",
+    "schema": ""
+  },
+  {
+    "display_name": "Latitude",
+    "to_field": "latitude",
+    "to_table_name": "PropertyState",
+    "from_field": "latitude",
+    "units": "",
+    "type": "float",
+    "schema": ""
+  },
+  {
+    "display_name": "Longitude",
+    "to_field": "longitude",
+    "to_table_name": "PropertyState",
+    "from_field": "longitude",
+    "units": "",
+    "type": "float",
+    "schema": ""
+  },
+  {
+    "display_name": "Longitude",
+    "to_field": "longitude",
+    "to_table_name": "PropertyState",
+    "from_field": "Longitude",
+    "units": "",
+    "type": "float",
```

Comment:
Entries like this (Longitude vs longitude) could be matched logically in the mapping_controller with `.toLowerCase()`

Reply:
That's a good point. I'm ok with this though to avoid some kind of unexpected match though it's pretty unlikely